demo/index.html

```diff
@@ -99,7 +99,7 @@
         }
         progress.value = progress.min;
       }
-      var animationFrame = requestAnimationFrame(nextProgress);
```

Comment:
@gsathya I don't even know why we keep a reference to `animationFrame` at all. Mind removing all references to `animationFrame`?